pkg/aws/v1alpha3/common_fetcher.go

```diff
@@ -122,13 +122,11 @@ func FetchCluster(m *Handler, meta metav1.Object) (*capiv1alpha3.Cluster, error)
 
 	// Fetch the Cluster CR
 	{
-		m.Logger.Log("level", "debug", "message", fmt.Sprintf("Fetching Cluster %s", clusterID))
+		m.Logger.Log("level", "debug", "message", fmt.Sprintf("Fetching Cluster %s in namespace %s.", clusterID, namespace))
```

Comment:
I added the namespace to logging and error so people will know why their request was denied.